src/mad_spark_alt/evolution/fitness.py

```
@@ -287,37 +207,26 @@
+            
+            for individual in population:
+                # Get novelty score from unified evaluation metadata
+                if "unified_scores" in individual.evaluation_metadata:
+                    novelty = individual.evaluation_metadata["unified_scores"].get("novelty", 0.0)
+                    total_novelty += novelty
+                    count += 1
+            
+            if count > 0:
+                avg_novelty = total_novelty / count
+                # High average novelty indicates diverse population
+                return avg_novelty
```

Ruff (0.12.2)

Unnecessary assignment to `avg_novelty` before `return` statement

Remove unnecessary assignment

(RET504)